apps/ade-cli/scripts/install-runtime.ps1

```
@@ -463,15 +547,40 @@ try {
+    # either is the machine's only copy -- an abort (Ctrl-C, a throw between
+    # the two renames) between moving the old one aside and moving the new one
+    # in would otherwise leave the machine with nothing installed.
     Remove-Item -LiteralPath $tempRoot -Recurse -Force -ErrorAction SilentlyContinue
+    Remove-Item -LiteralPath $pendingBinary -Force -ErrorAction SilentlyContinue
+    Remove-Item -LiteralPath $stagedRuntime -Recurse -Force -ErrorAction SilentlyContinue
+    if (-not $promotedRuntime -and
+        (Test-Path -LiteralPath $backupRuntime) -and
+        -not (Test-Path -LiteralPath $runtimeDir)) {
+      try {
+        Move-Item -LiteralPath $backupRuntime -Destination $runtimeDir -Force -ErrorAction Stop
+      } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)